lib/dcmd/arg.go

```diff
@@ -128,6 +128,36 @@ func (p *ParsedArg) Int64() int64 {
 	}
 }
 
+func (p *ParsedArg) Float64() float64 {
+	if p.Value == nil {
+		return 0
+	}
+	
+	switch t := p.Value.(type) {
+	case int:
+		return float64(t)
+	case int32:
+		return float64(t)
+	case int64:
+		return float64(t)
+	case float32:
+		return float64(t)
+	case float64:
+		return float64(t)
+	case uint:
+		return float64(t)
+	case uint32:
+		return float64(t)
+	case uint64:
+		return float64(t)
+	case string:
+		parsed, _ := strconv.ParseFloat(t, 64)
+		return parsed
```

Comment:
Remove this case for consistency with `ParsedArg.Int`: it's not `Float64`'s responsibility to parse string values, As is, this also discards any parse errors, which is inadvisable.
```suggestion
```